README.md

```diff
@@ -141,7 +142,7 @@ FRIENDS_PAGER="less -RF"
 - **Friends**: The people you do _activities_ with. Each friend has a name and,
   optionally, one or several nicknames. (Examples: `John`, `Grace Hopper`)
 - **Locations**: The places in which _activities_ happen. (Examples: `Paris`,
-  `Marie's Diner`)
```

Comment:
I think this is a good change regardless of the broader discussion. This example, however, intended to show that locations can be multiple words and contain punctuation. Could we switch to a relatively-known one that still fits that description? Maybe `Martha's Vineyard` or something similar? (I'm having trouble finding other examples, eek.)

Reply:
Coolio - in that case, keeping Marie's Diner as an example here is fine, no? If we are not worried about someone typing in `Went to _Marie's Diner_` and accidentally setting `Marie's Diner` as a default location (for the moment), then I'm happy to keep this example as is 👍

Reply:
I'm not totally sure I understand what you're saying. I think I'd like to change this to a geographic location that still uses both spaces and punctuation (`Martha's Vineyard`, for example) to show users that locations don't need to be a single word. Does that make sense?

I think I may be misunderstanding you—please correct me if so!

Reply:
Ah gotcha, this makes sense - I didn't know Martha's Vineyard was a place (I thought it might be a wine bar or something 😂) so happy to replace `Marie's Diner` with `Martha's Vineyard`. It's funny, I just tried Googling for a city/region with an apostrophe, and `Martha's Vineyard` kept coming up!